prov/verbs/src/fi_verbs.h

```diff
@@ -484,6 +484,17 @@ int vrb_srq_context(struct fid_domain *domain, struct fi_rx_attr *attr,
 int vrb_domain_xrc_init(struct vrb_domain *domain);
 int vrb_domain_xrc_cleanup(struct vrb_domain *domain);
 
+enum ibv_gid_type {
+	IBV_GID_TYPE_IB_ROCE_V1,
+	IBV_GID_TYPE_ROCE_V2,
+};
```

Comment:
This enum should be within an #if check, and not duplicated if already defined by verbs.h.  Also, IBV_GID_TYPE_IB should be the first value, so that we don't define an enum value with the same name, but a different value from verbs.h.